src/main/java/io/debezium/connector/db2/Db2Connector.java

```diff
@@ -74,10 +74,10 @@ protected void validateConnection(Map<String, ConfigValue> configValues, Configu
         ConfigValue hostnameValue = configValues.get(RelationalDatabaseConnectorConfig.HOSTNAME.name());
         // Try to connect to the database ...
         Db2ConnectorConfig connectorConfig = new Db2ConnectorConfig(config);
-        try (Db2Connection connection = new Db2Connection(connectorConfig.getConfig())) {
+        try (Db2Connection connection = new Db2Connection(connectorConfig.getJdbcConfig())) {
             try {
                 connection.connect();
-                connection.execute("SELECT version()");
+                connection.execute("SELECT 1 FROM sysibm.sysdummy1;");
```

Comment:
@jcechace, does this table always exist? Can't you just do `SELECT 1`?

Reply:
I'm not sure whether DB2 permits SELECT without a table. Anyway according to docs sysdummy1 should always exist.

https://www.ibm.com/docs/en/db2-for-zos/12?topic=tables-sysdummyx